src/components/RequestError.js

```diff
@@ -141,53 +125,66 @@ class RequestErrorModal extends React.Component {
    *
    * @param {Object} e Event emitted by the click
    */
-  showAdvanced = (e) => {
+  const showAdvanced = (e) => {
     e.preventDefault();
-    $(this.refs.advancedData).show(300);
-    $(this.refs.showAdvancedLink).hide();
-    $(this.refs.hideAdvancedLink).show();
+    $(advancedDataRef.current).show(300);
+    $(showAdvancedLinkRef.current).hide();
+    $(hideAdvancedLinkRef.current).show();
   }
 
   /**
    * Triggered when user clicks to hide advanced data
    *
    * @param {Object} e Event emitted by the click
    */
-  hideAdvanced = (e) => {
+  const hideAdvanced = (e) => {
     e.preventDefault();
-    $(this.refs.advancedData).hide(300);
-    $(this.refs.hideAdvancedLink).hide();
-    $(this.refs.showAdvancedLink).show();
+    $(advancedDataRef.current).hide(300);
+    $(hideAdvancedLinkRef.current).hide();
+    $(showAdvancedLinkRef.current).show();
   }
 
-  render() {
```

Comment:
tip: To review these changes to the `render()` method I suggest using the "Hide whitespace" feature on GitHub, as it highlights only the words that were changed on this file, and not the indentation adjustments necessary:

![Ignore Whitespace Diff](https://github.com/HathorNetwork/hathor-wallet/assets/1299409/1b603254-b05e-4698-80bb-c6360fd227c0)